tedana/utils/utils.py

```diff
@@ -133,17 +133,19 @@ def load_data(data, n_echos=None):
                              '{}'.format(data))
         else:  # individual echo files were provided (surface or volumetric)
             fdata = np.stack([load_image(f) for f in data], axis=1)
-            return np.atleast_3d(fdata), data[0]
+            fdata = np.atleast_3d(fdata)
+            ref_img = check_niimg(data[0])
+    else:
+        img = check_niimg(data)
+        (nx, ny), nz = img.shape[:2], img.shape[2] // n_echos
+        fdata = load_image(img.get_data().reshape(nx, ny, nz, n_echos, -1, order='F'))
 
-    img = check_niimg(data)
-    (nx, ny), nz = img.shape[:2], img.shape[2] // n_echos
-    fdata = load_image(img.get_data().reshape(nx, ny, nz, n_echos, -1, order='F'))
+        # create reference image
+        ref_img = img.__class__(np.zeros((nx, ny, nz)), affine=img.affine,
+                                header=img.header, extra=img.extra)
+        ref_img.header.set_sform(ref_img.header.get_sform(), code=1)
```

Comment:
This should probably be moved back next to `ref_img.header.extensions = []` so that it triggers regardless of the input data!